Add basic layout and logic

matheus-fls opened the pull request:
Adding basic app layout and logic to development branch. The app is currently stable and ready for first  production release.

matheus-fls committed:
Basic setup

matheus-fls committed:
Add basic layout and logic

matheus-fls committed:
saving for future reference, abandoning giphy API

matheus-fls committed:
Basic appearance ready

matheus-fls committed:
Add modules

matheus-fls committed:
Fix linter errors

matheus-fls committed:
Fix more errors

matheus-fls committed:
Add link to repo on page

matheus-fls committed:
Update index.html on src folder

matheus-fls committed:
Remove runtime import from index js, add babel config and minor visual improvements

matheus-fls committed:
Makes separate requests for C and F, add selector for C or F, refactor api.js, removes logic.js

matheus-fls committed:
Increase temperature font size

matheus-fls committed:
Fix linter error

matheus-fls committed:
Fix trailing space

matheus-fls committed:
Decrease top margin

matheus-fls committed:
Minor visual improvement

matheus-fls committed:
Add app image

matheus-fls committed:
Update README.md

matheus-fls committed:
Update README.md

Add live page link

matheus-fls committed:
Update README.md

matheus-fls committed:
Remove babel, fix border bug

matheus-fls committed:
Convert http to https on api request

Oghenebrume50 reviewed (approved):
STATUS: Changes Required

Hi @matheus-fls great work done here, nice implementation, :clap:  a few things to do now 

- change the background, when a search is done ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png)

- Output a better message, when a location is not found 

Please make changes and request for a re-review :slightly_smiling_face: 

Raphael Noriode :sunglasses:

matheus-fls committed:
Change alert message

matheus-fls committed:
Add icons for each weather condition

matheus-fls commented:
Thank you for your review Raphael!

I changed the alert message and added icons representing each weather condition. 

Submitting for another review.

Oghenebrume50 reviewed (approved):
STATUS: APPROVED

Hi @matheus-fls good job on the change made,  the icon code be bigger so the change is evident or maybe try to debug the code for the  image change. But that can be done later for now, this is approved  :+1: 

Raphael Noriode :sunglasses:
